src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Mappers/Apprentice/DetailsViewModelMapper.cs

```diff
@@ -25,6 +28,15 @@ public async Task<DetailsViewModel> Map(DetailsRequest source)
             var priceEpisodes = await _commitmentApiClient.GetPriceEpisodes(source.ApprenticeshipId);
             var detailsResponse = await detailsResponseTask;
 
+            var allowEditApprentice =
+                detailsResponse.Status == ApprenticeshipStatus.Live ||
+                detailsResponse.Status == ApprenticeshipStatus.WaitingToStart ||
+                detailsResponse.Status == ApprenticeshipStatus.Paused;
+
+            var editApprenticeURL = allowEditApprentice
```

Comment:
We have this helper in Url extension form, so you can use it in the razor directly, which is better than determining a url string property on the viewmodel, for example from `EditDraftApprenticeship.html` there's a link to cohort details:

```<a class="govuk-body govuk-link" href="@Url.ProviderApprenticeshipServiceLink($"{Model.ProviderId}/apprentices/{Model.CohortReference}/details")">Cancel</a>```